python/ray/autoscaler/aws/example-cloudwatch.yaml

```diff
@@ -0,0 +1,85 @@
+# An unique identifier for the head node and workers of this cluster.
+cluster_name: cloudwatch
+
+# Cloud-provider specific configuration.
+provider:
+    type: aws
+    region: us-west-2
+    availability_zone: us-west-2a
+    # Start by referencing Unified CloudWatch Agent json config file path here.
+    # Note that this config assumes Unified CloudWatch Agent is installed on every cluster node.
+    # For AMI with Unified CloudWatch Agent pre-installed, see https://github.com/amzn/amazon-ray.
```

Comment:
I think we can delete this, since it is mentioned more appropriately below above the `ImageId`.